config/robots.rb

```diff
@@ -1532,6 +1532,27 @@
       }
     )
 
+    # For scRNA Core pipeline cherrypick to 5p Dil plate
+    # equivalent of hamilton-lbc-cherrypick-to-lbc-5p-gex-dil in Bespoke 5p pipeline
+    custom_robot(
+      'hamilton-lrc-gem-x-5p-cherrypick-to-lrc-gem-x-5p-dil',
+      name: 'hamilton LRC GEM-X 5p Cherrypick => LRC GEM-X 5p Dil',
```

Comment:
About half the robot configs capitalise the Hamilton or Bravo in the name and half don't (also reflected in the integration suite tests). In file export names it mostly capitalised. It's a company brand name so I prefer capitals but have probably done it both ways myself and can be left to yet another 'tidy up the LIMS naming' story!